sci-physics/rivet/rivet-3.1.10.ebuild

```diff
@@ -36,7 +36,7 @@ RDEPEND="
 	>=sci-physics/fastjet-3.4.0[plugins]
 	>=sci-physics/fastjet-contrib-1.048
 	hepmc2? ( sci-physics/hepmc:2=[-cm(-),gev(+)] )
-	hepmc3? ( sci-physics/hepmc:3=[-cm(-),gev(+)] )
+	hepmc3? ( <sci-physics/hepmc-3.3.0:3=[-cm(-),gev(+)] )
```

Comment:
Modification of `RDEPEND` requires a revision bump.

Reply:
ah, it did no longer compile with 3.3.0, so I thought no bump is needed.

Reply:
An user who has it already installed with hepmc-3.2.0 will still be able to upgrade to 3.3.0, and presumably it will break then.

Reply:
Yes, that is what I meant with my second sentence.